.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,20 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/postgres
+{
+  "dockerComposeFile": "docker-compose.yml",
+  "forwardPorts": [3000, 5432],
+  "workspaceFolder": "/workspaces/human-essentials",
+  "service": "app",
+  "customizations": {
+    "vscode": {
+      "extensions": ["Shopify.ruby-extensions-pack"],
```

Comment:
Any other extensions we should be adding here?

Reply:
What does the Shopify extensions give you?

Reply:
This is the basic ruby extension for vscode. It includes a language server. This is probably the first extension someone would want to install for programming in ruby.